spec/services/activity_defaults_spec.rb

```diff
@@ -28,12 +29,50 @@
     let(:activity_defaults) do
       described_class.new(
         parent_activity: parent_activity,
-        partner_organisation: partner_organisation
+        partner_organisation: partner_organisation,
+        is_oda: is_oda
       )
     end
 
     subject { activity_defaults.call }
 
+    describe "`is_oda` and `transparency_identifier`" do
+      let(:parent_activity) { create(:programme_activity, :ispf_funded) }
+      let(:expected_transparency_identifier) { "#{Organisation::SERVICE_OWNER_IATI_REFERENCE}-#{subject[:roda_identifier]}" }
+
+      context "when activity is ODA" do
+        let(:is_oda) { true }
+
+        it { expect(subject[:is_oda]).to be true }
```

Comment:
In terms of consistency within this file, we've got a bit of a mix of `it { expect }` and `it "a human-readable test assertion"`. I personally prefer the latter, but am not too precious about it. I do think it'd be good to pick one style and be consistent with it throughout the file, though.

If you've got a preference for `it { expect }`, how would you feel about adding a separate commit to this PR to tidy up the rest of the `it`s for consistency where possible?

Otherwise, I think it would be good to stick with the other pattern. What do you reckon?

Reply:
Yep, no problem! Updated